src/firefly/js/util/WebUtil.js

```diff
@@ -467,7 +467,15 @@ export function parseUrl(url) {
 
     // Convert query string to object map with decoded key/value pairs
     const searchObject = {};
-    searchParams.forEach((val, key) => searchObject[key] = val);
+    const parseVal = (val) => {
+        try {
+            val = JSON.parse(val);
+        } catch(e) {
+            val = isBooleanString(val) ? toBoolean(val) : val;
+        }
+        return val;
+    };
+    searchParams.forEach((val, key) => searchObject[key] =parseVal(val));
```

Comment:
Good catch.  I see the code is taken from `decodeParams`, but it's wrong.  Please remove `decodeParams` function and change your `parseVal` to this:

```
    const parseVal = (val) => {
        try {
            val = JSON.parse(val);
        } catch(e) { /*ignore*/ }
        return isBooleanString(val) ? toBoolean(val) : val;
    };

```

Reply:
> Good catch. I see the code is taken from `decodeParams`, but it's wrong. Please remove `decodeParams` function and change your `parseVal` to this:
> 
> ```
>     const parseVal = (val) => {
>         try {
>             val = JSON.parse(val);
>         } catch(e) { /*ignore*/ }
>         return isBooleanString(val) ? toBoolean(val) : val;
>     };
> ```

@loitly Done as you suggested.  But I found this ticket was gone and replaced with a FF ticket. Are there any issues if I merge it to dev?

Reply:
Not a problem.  Just go and add this PR link to the new ticket, then merge when ready.

Reply:
OK, I will do the link and then merge this branch.  Thank you @loitly